src/tables/methods/createTableMethods.ts

```diff
@@ -222,7 +218,7 @@ export const createTableMethods = <PS extends Schema, M extends BaseTableMetadat
       return () => subscription.unsubscribe();
     }, [table, entity]);
 
-    return properties ?? defaultProperties;
+    return properties;
```

Comment:
why dont we return the default properties here?

Reply:
Seems redundant since we handle it in setState but can add back.